app/views/unattended/provisioning_templates/finish/windows_default_finish.erb

```diff
@@ -70,63 +65,75 @@ description: |
       <% end %>
     <% end %>
   <% end %>
+  
+  powershell /c "powercfg /setactive 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c"
```

Comment:
what is this UUID?

Reply:
this is the default Power Scheme GUID for High performance